CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,57 @@
+# Code of Conduct - Freedom of the Press Foundation
+
+Like the the broader technical and activism communities as a whole, the Freedom of the Press Foundation (FPF) team is made up of a mixture of staff and volunteers from all over the world, working on every aspect of our mission—including operations, software development, mentorship, and building connections with great people and organizations.
+
+To work together effectively in a large, diverse and open community, we have a few ground rules that we expect everyone to adhere to, be it FPF staff and board members, volunteers and event attendees; mentors, veterans, novices, or those seeking help and guidance.
+
+This isn’t an exhaustive list of things that you cannot do. Rather, take it in the spirit in which it’s intended—a guide to make it easier to enrich all of us and the communities in which we participate.
+
+This code of conduct applies to all spaces and events managed by the Freedom of the Press Foundation. This includes physical offices, GitHub repositories, online chat systems, the Support Forum, FPF hosted or sponsored events, and any other forums created by the Foundation which our team uses for communication. In addition, we take violations of this code outside these spaces into account when determining a person's ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you report it by talking to Emmanuel, or emailing **emmanuel@freedom.press**. If, for whatever reason, you are uncomfortable reporting the issue to Emmanuel, you may contact FPF’s Executive Director (**trevor@freedom.press**).  
```

Comment:
A few more suggestions:

- Add the pronouns for the people handling CoC  violation reports
- If @redshiftzero agrees, then please add her also into the team for handling reports. It had been pointed out before (in other cases/projects) that many times people prefer to report if only they find a diverse team handling CoC violation reports.

Reply:
I agree on having more than one person handling reports. Very valid feedback! I would be careful to volunteer any specific people unless they are willing to. (Thank you for understanding that in your comment here, @kushaldas :) )